README.md

```diff
@@ -7,17 +7,17 @@ libraries that have interesting bugs, hard-to-find code paths, or other
 challenges for bug finding tools.
 
 The current version supports [libFuzzer](http://libFuzzer.info),
-in future versions we exect to support [AFL](http://lcamtuf.coredump.cx/afl/)
+in future versions we expect to support [AFL](http://lcamtuf.coredump.cx/afl/)
 and potentially other fuzzing engines.
 
 # See also
 
 * [AddressSanitizer](http://clang.llvm.org/docs/AddressSanitizer.html)
 
 # Contributing
-See [CONTRIBUTING](CONTRIBUTING) first. 
-If you want to add one more benchmark to the test suite,
-simply mimic one of the existing benchmarks and send the pull request. 
+See [CONTRIBUTING](CONTRIBUTING) first.
+If you want to add one more benchmarks to the test suite,
```

Comment:
Either "add more benchmarks" or "add another benchmark" sounds better.

Reply:
Indeed, I missed the *one* beforehand. I improved the wording as you suggested.